index.html

```diff
@@ -1644,6 +1644,147 @@ <h4>Background</h4>
+<a href="https://en.wikipedia.org/wiki/Social_network">social
+networks</a>. By not directly controlling their own identifiers, people
+who want to use secure communication are not only unable to utilize
+cryptographic security in many of their everyday interactions, they are
+also limited to choosing between traditional secure messaging services
+to safely communicate with friends and family in a limited, chat-style
+context.
+        </p>
+        <p>
+In a traditional secure messaging approach the security is a property of the platform; if a user changes
+		platforms, either temporarily or permanently, they get their security a different way, tied to a
+		different login and different terms of service. DIDs take security out of the hands of the
+		context owner, i.e., an application or service provider, and put it in the hands of the DID
+		controller, who can choose the contexts in which they want to communicate and when. This makes
+		security decentralized and portable, instead of having the security of an interaction tied to
+		the protocol, transport, or application used to facilitate it. This feature allows people
+		to easily switch communication contexts without losing any security guarantees, and provides
+		a resilient backbone for all internet communications. Moving away from traditional identifiers
+		such as phone numbers and email addresses can help to break down the siloed ecosystems that
+		prevent data portability and make it difficult to more broadly incorporate secure communication 
+		technology in applications and services. Current approaches to secure communication
+		often have drawbacks such as the following:
+        </p>
+        <ul>
+          <li>reliance on commonly used identifiers which are not directly under user control</li>
+          <li>use of a central intermediary or centralized components to facilitate interactions</li>
+          <li>requirement that one or more parties have a highly available web server</li>
+          <li>siloed and non-interoperable security architectures tied to a single domain</li>
+        </ul>
+        <p>
+          DIDs help to solve this issue by enabling a style of communication which is truly peer-to-peer and
+		message-based. Message-based systems decouple the relationship between producers of messages
+		and consumers of messages, creating an abstraction that puts the two on equal footing and allows
+		for many different kinds of devices with differing requirements to interface in a flexible
+		and technology-neutral way. DIDs allow people to simultaneously use the same identifier for
+		both traditional communications and newer transactions that require a higher level of cryptographic security,
+		as well as allowing them to switch contexts without losing any security guarantees. These decentralized identifiers are not
+		rented or borrowed from centralized providers but are instead directly under user control.
```

Comment:
whitespace and line length
```suggestion
DIDs help to solve this issue by enabling a style of communication which
is truly peer-to-peer and message-based. Message-based systems decouple
the relationship between producers of messages and consumers of messages,
creating an abstraction that puts the two on equal footing and allows for
many different kinds of devices with differing requirements to interface
in a flexible and technology-neutral way. DIDs allow people to
simultaneously use the same identifier for both traditional communications
and newer transactions that require a higher level of cryptographic
security, as well as allowing them to switch contexts without losing any
security guarantees. These decentralized identifiers are not rented or
borrowed from centralized providers but are instead directly under user
control.
```